test.sh

```diff
@@ -447,8 +447,16 @@ sub=$(curl -sSf -u $SK: $HOST/v1/subscriptions \
            -d items[0][quantity]=5 \
       | grep -oE 'sub_\w+' | head -n 1)
 
+data=$(curl -sSf -u $SK: $HOST/v1/subscriptions/$sub \
+            -d items[0][plan]=annual-tiered-volume)
+
+same_data=$(curl -sSf -u $SK: $HOST/v1/subscriptions/$sub \
+                 -d items[0][plan]=annual-tiered-volume)
+
+diff  <(echo "$data" ) <(echo "$same_data")
```

Comment:
extra space here

Reply:
Indeed, thanks!
Here is a small diff of the fix:
```diff
--- a/test.sh
+++ b/test.sh
@@ -453,7 +453,7 @@ data=$(curl -sSf -u $SK: $HOST/v1/subscriptions/$sub \
 same_data=$(curl -sSf -u $SK: $HOST/v1/subscriptions/$sub \
                  -d items[0][plan]=annual-tiered-volume)
 
-diff  <(echo "$data" ) <(echo "$same_data")
+diff <(echo "$data") <(echo "$same_data")
 
 curl -sSf -u $SK: $HOST/v1/invoices?customer=$cus